docs/.vuepress/containers/sourceCodeLink.js

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
Because the basic markdown syntax could not allow us to style this link as we want. With this container, we can fully customize styles and behavior. 
It was also less time-consuming to do than any other solution.

Reply:
Same could be done with HTML anchor in Markdown. Simple coding, simple construction, better interoperability with other markdown engines and publishers.

Reply:
It doesn't hold back the initial release. Let's do it at your earliest convenience @Siemienik

Reply:
Reported https://github.com/handsontable/handsontable/issues/8424.

Reply:
Reported https://github.com/handsontable/handsontable/issues/8424.